ansible/.generate/defaults_main.yml

```diff
@@ -0,0 +1,2 @@
+# Update file in operator-sdk-samples/ansible/generate/defaults_main.yml
```

Comment:
should this comment be here?

Reply:
This is here because when it gets moved into the memcached operator. I figured if someone wants to make a change to the sample, they might try to do it directly, but instead they should change the sample generation to make it reproducible.